Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -225,7 +225,12 @@ def print_caveats
   def enable_accessibility_access
     return unless @cask.accessibility_access
     ohai "Enabling accessibility access"
-    if MacOS.version <= :mountain_lion
+    if MacOS.version >= :sierra
```

Comment:
Done. Now I'm wondering if the two `disable` messages should just be combined? 

<= Mountain Lion: 
```
        Accessibility access was enabled for #{@cask}, but it is not safe to disable
        automatically on this version of macOS.  See System Preferences.
```
Sierra: 
```
        Accessibility access cannot be disabled automatically on this version of macOS.
        See System Preferences to disable it manually.
```